Layouts/layout-details-Access.json

```diff
@@ -1,5 +1,7 @@
 {
 	"typeId": "Access",
+	"releaseNotes": "Apply incident source fields",
+	"fromVersion": "4.5.0",
```

Comment:
if you are adding fromVersion then you need a copy if the json with toVersion 4.0.0

Reply:
Right, will change.